src/clients/soap/utils.ts

```diff
@@ -96,16 +124,19 @@ const overrideClientDefaults = (
   }
 };
 
+/**
+ * Make a request to a SOAP-based API
+ * @param param0 An object containing how to connect to a SOAP-based API, the method to onvoke, any overridden values for the HTTP client, and whether or not to log debug messages
```

Comment:
Would it be better to move this part of the docs to the `RequestParams` type itself? I think that would provide better in-editor context though I'm unsure about the actual doc generation tooling.

Reply:
I think you're right - you can command-click the argument to get more detailed info about each destructured element.